bgpd/bgp_snmp.c

```diff
@@ -29,10 +29,108 @@
 #include "bgpd/bgp_snmp_bgp4.h"
 #include "bgpd/bgp_snmp_bgp4v2.h"
 #include "bgpd/bgp_mplsvpn_snmp.h"
+#include "bgpd/bgp_snmp_clippy.c"
+
+uint32_t bgp_snmp_traps_flags;
+
+static int bgp_cli_snmp_traps_config_write(struct vty *vty);
+
+
+DEFPY (bgp_snmp_traps_rfc4273,
+       bgp_snmp_traps_rfc4273_cmd,
+       "bgp snmp traps rfc4273<enable$on|disable$off>",
+       BGP_STR
+       "Configure BGP SNMP\n"
+       "Configure SNMP traps for BGP\n"
+       "Configure use of rfc4273 SNMP traps for BGP\n"
+       "Specify using of rfc4273 traps\n"
+       "Specify unusing rfc4273 traps\n") 
+{
+	if (on) {
+		SET_FLAG(bgp_snmp_traps_flags,
+				BGP_SNMP_TRAPS_RFC4273_ENABLED);
+		return CMD_SUCCESS;
```

Comment:
We can move return to the end of the function, instead of returning in every if block.

Reply:
done